### PR TITLE
New Feature buzzer

### PR DESCRIPTION
**Pull request by mayank-kabra2001**
Fixes #

#### Describe the changes you have made in this PR -

Created a new feature of buzzer - 

Inputs are frequency of the sound , duration upto which sound is played , and a play button . 
eg . input of 426 as frequency and 2 as duration and 1 as play will produce a C chord(426 Hz) for 2sec if play input is 1.

In properties its volume can be adjusted .
Also whether we want that frequency (eg. C chord) to be sharp or flat (i.e. C# or C') can be controlled by a checkbox .  

### Screenshots of the changes (If any) -

Screenshot from 2020-12-17 02-09-28

Note: Please check **Allow edits from maintainers.** if you would like us to assist in the PR.

### CODE REVIEW

File: public/src/ux.js [hunk 1]
```diff
@@ -233,7 +233,7 @@ export function showProperties(obj) {
     if(layoutModeGet()){
         // if an element is selected, show its properties instead of the layout dialog
         if (simulationArea.lastSelected === undefined || ['Wire', 'CircuitElement', 'Node'].indexOf(simulationArea.lastSelected.objectType) !== -1){
-            $('#moduleProperty').hide();
+            $('#modulePropshowPropertieserty').hide();
```
Comment (abstrekt) [inline public/src/ux.js:236]: is this a mistake?

File: public/src/moduleSetup.js [hunk 2]
```diff
@@ -43,6 +43,7 @@ import TwoComplement from './modules/TwoComplement';
 import VariableLed from './modules/VariableLed';
 import XnorGate from './modules/XnorGate';
 import XorGate from './modules/XorGate';
+import Diode from './modules/Diode' ;
```
Comment (abstrekt) [inline public/src/moduleSetup.js:46]: trouble with webpacker due to this import. might have mixed code with #2045 ?

**Review by abstrekt — CHANGES_REQUESTED** [review]
I was testing this feature out, there some to be some unrelated codes maybe relating to PR #2045, I'm not sure if this feature is working as it suppose to in a normal use case but there is a bug where the buzzer won't stop -- to reproduce try playing the buzzer without providing any input in for the duration.

Seems like a great feature --would love to see this through, can you fix the above mention issues so that @satu0king can review it to be working properly?

**Comment by gr455** [comment]
This PR still needs work. A bunch of bugs with this one

**Comment by tachyons** [comment]
Closing due to inactivity, feel free to re-open once the issues are addressed

**Pull request closed without merging**